src/modules/models/types.ts

```diff
@@ -0,0 +1,13 @@
+export type ModelMetrics = {
+  triangles: number
+  materials: number
+  meshes: number
+  bodies: number
+  entities: number
+  textures: number
+}
+
+// TODO import Vector3 and Quaternion from a library ??? There are multiple implementations of it (decentraland-ecs, babylon, three)
+export type Vector3 = { x: number; y: number; z: number }
```

Comment:
What about all the different Vector3 and Quaternion types that are lying around?

Reply:
I _think_ we can import those from `decentraland-ecs`

Reply:
After further analysis, it seems that the types that come from `decentraland-ecs` define classes, so it seems that we either stick to these values or change wherever we use them to create instances of those clases.